### ISSUE TITLE
The package can't be used in any Python 3.12 version greater than 3.12.0

### ISSUE THREAD

**Issue by bmitc** [cyrilou242/mockseries#52]
This line is overly restrictive:

https://github.com/cyrilou242/mockseries/blob/806e846afb59e83a39d0fa1da4fc61838705ad36/setup.py#L47

It prevents `mockseries` from being used in any Python 3.12 version that is not 3.12.0. So even 3.12.1 or 3.12.2 won't work since they do not satisfy the `<=3.12` constraint. I think what should be there is actually `<3.13`.

I am using Poetry to manage my project(s), and this is the error I got when I tried `poetry add mockseries` on my project, where I currently have Python 3.12.1 installed, which means that changing my `pyproject.toml` to say `>=3.9,<=3.12` would fail.

```
The current project's supported Python range (>=3.9,<3.13) is not compatible with some of the required packages Python requirement:
  - mockseries requires Python >=3.9,<=3.12, so it will not be satisfied for Python >3.12,<3.13

Because no versions of mockseries match >0.3.0,<0.4.0
 and mockseries (0.3.0) requires Python >=3.9,<=3.12, mockseries is forbidden.
So, because control-server depends on mockseries (^0.3.0), version solving failed.

  • Check your dependencies Python requirement: The Python requirement can be specified via the `python` or `markers` properties

    For mockseries, a possible solution would be to set the `python` property to ">=3.9,<=3.12"

    https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies,
    https://python-poetry.org/docs/dependency-specification/#using-environment-markers
```

**Comment by cyrilou242** [comment]
interesting thanks, let me fix this

**Comment by cyrilou242** [comment]
Hey @bmitc this should be fixed in the version `0.3.1`
